e2e_testing/step_time_bounds.yaml

```diff
@@ -48,6 +48,13 @@ benchmarks:
     confidence_interval: 0.06374
     average: 3.2879
     sample_size: 47
+  llama-3-8b-ddp-fsdp: # TODO(rupliu): to update
```

Comment:
is the name correct?
also please remove todo

Reply:
cool, updated!